pkg/aws/client.go

```diff
@@ -63,3 +65,9 @@ func NewClientFromEC2(ec2 EC2Client) *Client {
 		ec2: ec2,
 	}
 }
+
+func NewClientFromSnowball(snowballdevice SnowballDeviceClient) *Client {
+	return &Client{
```

Comment:
If we are doing something like this instead, is it possible to separate the structs for `Client` and do the following?

```
type Client struct {
   *EC2Client
   *SnowballDeviceClient
}
```
That way, we can have snowball and ec2 related member methods separately, and still utilize a common Client if needed by the consumers. However, I am just throwing an idea and not sure if it works for this case.